### ISSUE TITLE
Import Error while running lazydocs

### ISSUE THREAD

**Issue by RenukaMane** [ml-tooling/lazydocs#36]
**Describe the bug:**

While trying to run it against the python directory I am getting an import error for get_terminal_size. Is there any way to sort out this error. 

**Error Screenshot**

<img width="944" alt="lazyDocs error" src="https://user-images.githubusercontent.com/53396422/162428348-ede5373d-17b9-4260-ad9f-386400fd9f35.png">

**Technical details:**

- Host Machine OS (Windows/Linux/Mac): Windows
- Browser (Chrome/Firefox/Safari): Chrome

**Possible Fix:**

**Additional context:**

**Comment by mane292** [comment]
I also encountered this issue. After poking around a bit I believe that lazydocs only recognizes modules starting with a letter alphabetically before the current module. For example, when I tried to document DBpipeline, ILpipeline, and FPutils classes (in which DBpipeline and ILpipeline classes are dependent on FPutils), lazydocs did NOT find FPutils when trying to document DBpipeline, but recognized ILpipeline after documenting FPutils.
